tests/error/compiletime_error_closing_quote_expected/test_identity.nim

```diff
@@ -1,5 +1,5 @@
 import std/unittest
-import hello_world
+import identity
```

Comment:
The `run-tests.sh` script passes in the exercise slug, and all test cases use "identity" except for this one and a couple of others. I could have added some logic to the test runner to figure out the slug, or just renamed it (which is what I did).